src/Command/RunCommand.php

```diff
@@ -159,10 +163,30 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $job->getConfigRowIds(),
                 $outputs
             );
+
             $result = new JobResult();
             if (count($outputs) === 0) {
                 $result->setMessage('No configurations executed.');
             } else {
+                $outputTables = new TableCollection();
+                $inputTables = new TableCollection();
+                foreach ($outputs as $output) {
+                    $tableQueue = $output->getTableQueue();
+                    if ($tableQueue) {
+                        foreach ($tableQueue->getTableResult()->getTables() as $tableInfo) {
```

Comment:
Ten helper možná mohl konvertovat rovnou tu tableQueue, ještě se na to potom podívám, budu do toho ještě rýpat